### PR TITLE
Don't show creds for shared models

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #3426.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- load the new profile
- click on a shared model row
- it should not expand

**Comment by huwshimi** [comment]
[Don't show shared creds](https://trello.com/c/XkGTEdim/217-dont-show-shared-creds)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8428/

**Comment by hatched** [comment]
@huwshimi  looks like this is conflicted now.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8435/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8436/

**Comment by huwshimi** [comment]
This is strange, lint is failing on a file that I haven't modified and the linter passes locally.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8437/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8438/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8439/

File: jujugui/static/gui/src/app/components/deployment-flow/sshkey/sshkey.js [hunk 1]
```diff
@@ -73,11 +73,12 @@ class DeploymentSSHKey extends React.Component {
         };
       }
     }
-    this.setState({error: (<span>Key is invalid. Please
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/deployment-flow/sshkey/sshkey.js]: These are all drive-by fixes due to CI complaining.

File: package.json [hunk 2]
```diff
@@ -43,8 +43,8 @@
     "browserify-incremental": "^3.1.1",
     "chai": "4.1.1",
     "diff": "3.3.0",
-    "eslint": "^4.15.0",
-    "eslint-plugin-react": "^7.5.1",
+    "eslint": "4.15.0",
```
Comment (huwshimi) [inline package.json]: Attempt to get CI to stick to the right version of eslint.

**Comment by huwshimi** [comment]
@hatched I've updated this to show the message.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8445/

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**